etc/netatalk/netatalk.c

```diff
@@ -369,18 +369,14 @@ static void show_netatalk_version( void )
 
 static void show_netatalk_paths( void )
 {
+	printf( "              afp.conf:\t%s\n", _PATH_CONFDIR "afp.conf");
 	printf( "                  afpd:\t%s\n", _PATH_AFPD);
 	printf( "            cnid_metad:\t%s\n", _PATH_CNID_METAD);
-
-#ifdef HAVE_TRACKER
-	printf( "       tracker manager:\t%s\n", TRACKER_PREFIX "/bin/" TRACKER_MANAGING_COMMAND);
+  
+#ifdef WITH_SPOTLIGHT
```

Comment:
Isn’t dbus strictly speaking used for other functionality too, even if you’re not building spotlight? E.g. afpstats

Reply:
Well spotted. I'll look into it as just moving bus-daemon path outside of the Spotlight conditional causes compilation failure.
dbus-session.conf is Spotlight only.